src/renderer/components/reader/ReaderHeader.tsx

```diff
@@ -54,6 +54,8 @@ interface Props extends TranslatorProps {
 export class ReaderHeader extends React.Component<Props, undefined> {
     private enableFullscreenRef: any;
     private disableFullscreenRef: any;
+    private catchSettingFocus: any;
+    private catchMenuFocus: any;
```

Comment:
To avoid using `any`, could we instead declare the type of the private members by assigning them a proper "ref", for example `private catchMenuFocus = createRef<HTMLElement>();` (`import { createRef } from 'react'`), then instead of `ref={(ref) => this.catchMenuFocus = ref}` in the JSX DOM, we could simply use `ref={this.catchMenuFocus}`, and the usage of the private member can be safeguarded by checking for undefined/null `current` value, e.g.: `if (this.catchMenuFocus.current) { this.catchMenuFocus.focus(); }`. What do you think?